### DISCUSSION TITLE
[Feature Request] Skeleton component for loading content.

### DISCUSSION THREAD

**Discussion by lacymorrow** [heroui-inc/heroui#981] [Feature requests]
### Is your feature request related to a problem? Please describe.

I would love for a generic skeleton to be included as a component for this library for loading text content, blogs, etc.

I see that the image component has one built in, but I didn't see anything for text or content directly.

### Describe the solution you'd like

As a user, I would like a generic component for adding a skeleton loading animation for content. Optionally with customizable parameters for line length, mock image, etc.

### Describe alternatives you've considered

Including a skeleton from another library

**Comment by jrgarciadev** [comment]
Stay tuned 🔥
